java/jsinterop/generator/closure/visitor/MemberCollector.java

```diff
@@ -60,7 +60,7 @@ public void accept(TypedScope scope) {
     super.accept(scope);
 
     checkState(
-        parameterNameMapping.isEmpty(), "Unused parameter name mapping: %s", parameterNameMapping);
```

Comment:
Probably should revert these whitespace changes - not sure if there is a style guide or checker in place that can catch these (or format before merge)

Reply:
i will fix this, i dont know where they come from since i didnt touch this line, maybe some formatting thingy

Reply:
> not sure if there is a style guide or checker in place that can catch these (or format before merge)

https://github.com/google/google-java-format

I'm not aware of any integration with Bazel though, so use your IDE or the command-line.